### PR TITLE
Add provider to parameters

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java [hunk 1]
```diff
@@ -80,9 +79,9 @@ public SensitivityAnalysisService(@Value("${sensitivity-analysis.default-provide
         this.objectMapper = Objects.requireNonNull(objectMapper);
     }
 
-    public UUID runAndSaveResult(UUID networkUuid, String variantId, String receiver, String provider, ReportInfos reportInfos, String userId, UUID parametersUuid, LoadFlowParametersValues loadFlowParametersValues) {
+    public UUID runAndSaveResult(UUID networkUuid, String variantId, String receiver, String provider, ReportInfos reportInfos, String userId, UUID parametersUuid, UUID loadFlowParametersUuid) {
 
-        SensitivityAnalysisInputData inputData = parametersService.buildInputData(parametersUuid, loadFlowParametersValues);
+        SensitivityAnalysisInputData inputData = parametersService.buildInputData(parametersUuid, loadFlowParametersUuid);
 
         SensitivityAnalysisRunContext runContext = new SensitivityAnalysisRunContext(networkUuid, variantId, inputData, receiver, provider, reportInfos, userId);
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisService.java:86]: you cannot use 'provider' here, it is nnot sent by the Front.
Like in SA, SensitivityAnalysisRunContext has to read entity to get the provider (and maybe use default provider if missing)

File: src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java [hunk 2]
```diff
@@ -101,9 +100,9 @@ public ResponseEntity<UUID> runAndSave(@Parameter(description = "Network UUID")
                                            @Parameter(description = "reporterId") @RequestParam(name = "reporterId", required = false) String reporterId,
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java:100]: Line 98 you have to remove provider param (not sent by the Front)

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: things to change with provider managment in runContext

test: an issue when we have specific LF param (du to null provider) 
-> Sensitivity analysis provider not found null

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**